src/lib/Scenes/Fair/Components/FairHeader/index.tsx

```diff
@@ -88,7 +88,7 @@ export class FairHeader extends React.Component<Props, State> {
       <>
         <EntityList
           prefix="Works by"
-          list={artistList}
+          list={fairfollowedArtistList.concat(artistList)}
```

Comment:
```var a1 = [1, 2, 3]
var a2 = [3, 2, 1]
a1.concat(a2)
(6) [1, 2, 3, 3, 2, 1]```

I dont think we want to concat the two arrays because there could be duplicate names and it could display the same name twice.

Reply:
Try something like this

```
const concatArray = fairfollowedArtistList.concat(artistList)
const uniqArray = concatArray.filter((v,i) => concatArray.indexOf(v) == i)

...

list={uniqArray}
```

Reply:
@orta Do you think this is heavy for the client? Should we move this to it's own MP field?

Reply:
Nah, I think this level is fine enough - good call on making sure it's unique :

Reply:
( Mainly because I can see the usefulness in having two separate queries for this data )

Reply:
good catch!!! yes makes total sense @kierangillen